var/spack/repos/builtin/packages/metis/package.py

```diff
@@ -17,13 +17,15 @@ class Metis(Package):
        multilevel recursive-bisection, multilevel k-way, and multi-constraint
        partitioning schemes."""
 
-    homepage = "http://glaros.dtc.umn.edu/gkhome/metis/metis/overview"
-    url      = "http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/metis-5.1.0.tar.gz"
-    list_url = "http://glaros.dtc.umn.edu/gkhome/fsroot/sw/metis/OLD"
+    #
+    # the previous metis website http://glaros.dtc.umn.edu/gkhome/metis/metis
+    # no longer exists. This is a github mirror that provides metis 5.1.0
+    #
+
+    homepage = "https://github.com/scivision/METIS/"
+    url      = "https://github.com/scivision/METIS/raw/master/metis-5.1.0.tar.gz"
```

Comment:
I wish there were a better permanent solution. Only having access to a single version is kind of sad.

Reply:
I agree, but I think this is the only option. The other site is completely gone. It has been gone for a bit

Reply:
Do we want to do the same thing to the `parmetis` package?

Reply:
Good call, we aren using parmetis and I am not an expert on testing it. 

I added a spack issue to help track the need for url update:
https://github.com/spack/spack/issues/13762